File: src/main/java/org/profit/util/PathUtils.java
package org.profit.util;

import java.io.File;

/**
 * Path utilities
 *
 * @author Kyia
 */
public class PathUtils {

	private static String rootPath;

	private static final String CONF_PATH = File.separator + "conf" + File.separator;
	private static final String WEBROOT_PATH = File.separator + "webroot" + File.separator;
    private static final String LOCALE_PATH = File.separator + "locale" + File.separator;

	static {
		rootPath = new File("").getAbsolutePath();
	}

	public static String getRootPath() {
		return rootPath;
	}

	public static String getConfPath() {
		return getRootPath() + CONF_PATH;
	}

	public static String getWebRootPath() {
		return getRootPath() + WEBROOT_PATH;
	}

    public static String getLocalePath() {
        return getRootPath() + LOCALE_PATH;
    }
}

File: src/main/java/org/profit/util/PathUtils.kt
package org.profit.util

import java.io.File

object PathUtils {

    var rootPath: String? = null

    private val CONF_PATH = File.separator + "conf" + File.separator
    private val WEBROOT_PATH = File.separator + "webroot" + File.separator
    private val LOCALE_PATH = File.separator + "locale" + File.separator

    val confPath: String
        get() = rootPath + CONF_PATH

    val webRootPath: String
        get() = rootPath + WEBROOT_PATH

    val localePath: String
        get() = rootPath + LOCALE_PATH

    init {
        rootPath = File("").absolutePath
    }
}